graphql/schema.graphql

```diff
@@ -369,6 +370,12 @@ input CopyProjectInput {
   newProjectId: String
 }
 
+input MoveProjectInput {
+  id: String!
+  owner: String!
+  repo: String!
```

Comment:
can we maybe label these newOwner and newRepo so that it's clear that these are the values of the repo we will be moving it to?